src/main/resources/music/settings/host/values/strings.xml

```diff
@@ -20,18 +20,18 @@
     <string name="revanced_change_start_page_entry_home">Home</string>
     <string name="revanced_change_start_page_entry_library">Library</string>
     <string name="revanced_change_start_page_entry_subscription">Subscriptions</string>
-    <string name="revanced_change_start_page_summary">Changes the start page of the app.</string>
+    <string name="revanced_change_start_page_summary">Select which page the app opens in.</string>
```

Comment:
I wanted to make this summary match the patch description, but there is an issue.

The new patch description is `Adds an option to set which page the app opens in instead of the homepage.`

But "instead of the homepage" wouldn't make sense in the summary because one of the radio buttons the user can select is `Homepage`. Therefore this summary only partially matches the patch description.